lib/index.coffee

```diff
@@ -30,9 +41,10 @@ lint = (editor) ->
     throw new Error stderr or stdout unless typeof parsed is 'object'
     (parsed.files?[0]?.offenses or []).map (offense) ->
       {cop_name, location, message, severity} = offense
+      {cop_name, message} = extractUri cop_name, message
       {line, column, length} = location or DEFAULT_LOCATION
       type: if WARNINGS.has(severity) then 'Warning' else 'Error'
-      text: (message or DEFAULT_MESSAGE) +
+      html: (message or DEFAULT_MESSAGE) +
```

Comment:
You need something to escape any HTML already in the message (`escape-html` package is used in other linters for this purpose). `extractUri` would be a good place to put this (and a reason for it to return `message` :wink:).